tests/smoke_tests/monitors_test/postgres_test.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2014-2020 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import absolute_import
+
+if False:  # NOSONAR
+    from typing import Dict
+    from typing import Any
+
+import time
+import os
+
+import pytest
+
+from scalyr_agent.third_party import pg8000
+
+from tests.utils.agent_runner import AgentRunner
+
+from tests.utils.dockerized import dockerized_case
+from tests.image_builder.monitors.common import CommonMonitorBuilder
+from tests.utils.log_reader import LogMetricReader, AgentLogReader
+
+import six
+
+HOST = "localhost"
+USERNAME = "scalyr_test_user"
+PASSWORD = "scalyr_test_password"
+DATABASE = "scalyr_test_db"
+
+
+@pytest.fixture()
+def postgresql_client():
+    os.system("service postgresql start")
+
+    os.system(
```

Comment:
I know this code is only used in the test context and we use hard-coded values, but still, to be on the safe side and to avoid various static analysis tools to complaining, we should quote all the arguments which get passed to those commands - https://docs.python.org/3/library/shlex.html#shlex.quote (aka username, password).

Reply:
Thanks for the good advice. Done.